nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -129,6 +129,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - The `power.ups` module now generates `upsd.conf`, `upsd.users` and `upsmon.conf` automatically from a set of new configuration options. This breaks compatibility with existing `power.ups` setups where these files were created manually. Back up these files before upgrading NixOS.
 
+- `pdns` was updated to version [v4.9.0](https://doc.powerdns.com/authoritative/changelog/4.9.html#change-4.9.0), which introduces breaking changes. Check out the [Upgrade Notes](https://doc.powerdns.com/authoritative/upgrading.html#to-4-9-0) for details.
```

Comment:
You might better future-proof this by saying `4.9.x`, because I'm sure we won't remember to revisit this if we make a further bump before release.